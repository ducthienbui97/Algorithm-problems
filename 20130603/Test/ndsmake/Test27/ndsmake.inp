2803
-356545735 -283175969 -286305845 -457210897 75652102 -824032325 4196786 -96865200 919658956 715934972 826500702 913384226 790166757 807210149 996907051 -471174757 -202111087 854941909 -168688361 127008431 585933146 -176150093 -115197246 225488616 -868558776 -361187935 21561210 886387321 444319390 -189205564 571179486 -555618999 -588348015 491828695 94492641 -630663468 -936879852 20120716 389229550 -720421473 -999913778 876875930 395074086 -214955946 -279352787 359721576 -151789628 578989700 -399804524 -928855468 619323042 -361606240 -32011893 701541352 -887366663 972478578 -691490012 -707940109 -251895386 563613556 -320538492 122948739 -143689142 523512758 584642396 80608543 669066325 712427290 677384887 -427476210 82511117 -433976372 -105932523 607354223 822295860 -465044319 423252518 -636158362 876322391 -288579175 653420068 885641329 561162931 988232323 186740941 34469592 -3905084 -933709376 -641920516 -811773117 -677397653 854583583 -867863621 181910248 -702630169 495559126 625027424 473065716 -970228968 -687572737 71319724 -389274631 -688844980 192929582 -785752769 -648399644 -328363561 508348618 153551955 988254815 173973224 -541735520 -537994846 -19002254 179597443 -248104357 -637304588 816533573 688702486 -921796845 818925128 -713873916 -309837893 92935420 937594526 481585883 -780226295 -80853697 -820980897 979134752 910303127 -589653275 -319731094 60575889 249974031 -422983184 -563032900 -368136014 -914236182 611890564 735636923 678885209 518154050 -138209979 439160448 -816125514 -904426776 435574919 12589030 474747250 246431557 -338258933 204079465 543466228 113949526 780913672 398704963 -901930233 841772293 -690080963 -383450761 413126257 -234878409 -353224736 13924912 -703931212 134829401 -637594108 -137504534 -980226713 137498204 52132832 677838751 -320648552 943483436 887293269 935138114 -234641892 -616755193 631464995 -452763306 -564245353 538381666 231217714 933799694 -553054977 663069245 -965857886 86414870 67266703 -475944568 111541895 876115165 935230848 359625864 360377099 -852622686 353415754 -497293841 -383232360 723032102 -512778592 397087489 887482478 62124562 -100407623 49117353 -181250182 1037679 -969736868 368244897 242432641 165401383 569325761 -866131515 -489299419 370641401 936686604 621055232 -656180802 733515367 730109798 -677870615 840730823 84853178 68578144 940442377 352796003 -397120758 431811330 -589047812 -983708291 493878891 -93835911 -631357251 451167841 -732528477 688349600 -637691200 -61329761 14450781 48564145 716234680 -683334648 -871683833 -590883816 -382278398 303962070 -246159439 554887912 -615594207 -790931626 -116420810 915688490 786766751 607288604 -569888596 846453523 469241914 -240587624 615169214 -869486520 -271788082 -477694760 -697605578 373329502 994721075 -855639637 -577663198 -606593860 -378486405 -805867959 191535780 -82268513 -125557817 -860518095 -480181929 -603441096 -558936890 -507895024 -550846886 -578800656 -116305187 480242426 -910534197 -223276150 91451136 -561108246 -905874320 842763188 781855635 413430571 964563612 -431122845 491367116 -455736597 47000957 620527181 817757653 700252907 646270729 -702071251 -888728644 -782998448 965331095 406776594 177763034 -982053180 771482457 -50465718 383442650 575180475 926178938 233189501 -231860888 153822672 -979864368 -57579051 71396216 -771539123 313080667 453817648 -851023159 483397801 -996789923 -73432461 399777516 649034901 -37535989 -183281141 -463900325 -858670302 936263026 -312559490 -174649079 -898943552 -594714819 -298163937 -799298602 698456405 -167471578 -624953556 -513191104 572729530 -740444492 119801435 676152516 -770765732 657169801 269298848 213255737 537936017 -912829368 732892616 858303946 -777005301 -699770079 987078470 -138907762 -894657174 209699066 -820367861 507194054 -944416021 -531213253 -764436338 977532734 -510077386 -95840533 544415504 -5571466 -162153678 -973460697 420661298 -540277222 921608467 -425970798 330482363 528797528 861947559 186386757 -39511306 -523652519 679898511 986641611 155992018 -641758544 -726945592 78217948 -147102338 34868459 -76809771 178725701 -26462307 -915225268 -630137205 343512035 610668434 -713436808 943503821 -925388837 425097712 85372657 -338449774 -265680540 -489675489 -820391753 -136460392 469919752 -706948209 -771822173 602651911 -197494344 115003321 -39593354 572692118 -845069936 896928092 547097236 556586871 105452309 492336592 -973836001 344395317 713937929 77049045 563252194 595445241 -518868802 -225299272 -129665686 34917132 -113778681 624385112 -897927346 633601258 611216342 362725008 175171583 588573636 976745944 93123612 -922353407 -282832461 684110714 -93217939 -213327121 234484947 -801130161 -457746289 -620738709 -335917025 897541311 576970465 351179495 31054222 -203619346 -375823169 -155164985 530828712 -743958684 925947766 349925119 467417179 -59222718 -303684904 -102271957 -609303901 43644576 760426485 -805331781 462713044 -691128043 -82193032 880199500 -559236217 -763677100 878064543 -626833391 871566623 632706035 189830616 -414952592 416828919 -389632696 -740147287 810076458 -336529044 231525644 -919867471 615812961 -793627154 -412401171 389630544 -941910718 514602810 -248688814 128462826 516122429 310713657 -259502931 -844834063 -118750465 178272647 -560794478 708561871 -165577843 -688440261 299169332 377086308 -833726873 65467049 963108490 547576797 316068294 949766675 -639292900 331672511 -324004748 619285375 -427333268 -281055484 -433831609 -742767964 35069532 587860027 -686726658 640267799 307825812 342119937 288144598 401996308 761190762 540267857 572497538 630742536 648391301 -621053104 376566426 -978707698 -970857564 -480315737 -751348758 -536087335 169270348 -761681269 -411718601 711039400 -107288080 370953814 -7257640 -386561135 98638667 306629081 511650556 -525482714 336335041 904815380 -294395433 77446851 559069411 -482947694 470919671 -77119881 532484882 -333015713 607041717 -368575347 -323012037 86835111 -124243258 692635318 560163796 -699052425 198400228 63315419 -549496219 -703605301 -994137106 220454654 -59147382 -723170048 -844381650 -386966856 -511073643 25466956 -664597120 857816002 921714116 -460750797 -920028488 663597089 -320191253 -271937140 -359820361 771710313 -396953472 -333011789 657673910 -204655276 -147386133 -897259796 -632808574 -452442205 860832533 593165936 -279639209 483830328 12821219 -842558375 395507939 -881229472 -182563914 387245811 -753137100 841588261 -855976002 -386834499 -582637292 307473752 710542114 -954843447 120004180 -298008577 -669869445 -753196301 924947695 -613069853 -809776574 196398982 915563765 570011039 -323334348 -2300240 600817146 405075507 378568922 -822315511 3323024 688146511 -198510255 188476025 952330265 -450334271 872201717 -488705705 741272496 -414504642 -434428192 715828167 -43669156 267659343 332768878 993941702 -175318081 -109175288 643317211 -919340402 31843193 -377660064 -430575921 348702919 -77500785 -807527605 820780668 -910405973 390075735 475554358 770796222 475588996 25555888 -447935930 -400771705 214593113 424142178 524319504 432979215 -506928327 168617635 467504881 -58211032 -412531930 577616566 -184741364 -240827473 98981537 -983754065 -113510690 999334430 722094799 -257634735 -960018881 -92122811 -330813160 863769208 993342459 -395300235 -909982382 376020539 861846097 -224240460 664901479 -674171325 350928226 -157705753 -673164486 -597274712 -980666934 836701608 394503200 -843879353 512294620 -908860370 542152366 -425737092 911813611 124499111 488479879 728158263 234835762 78968487 -169311821 979249222 108438578 895244236 184444027 823830851 734629999 711613166 -110264535 -178954988 -819590823 354592604 -346643313 572044535 306023277 -13501324 -161517592 852925655 154651825 389873407 -604598231 -943703759 677827929 -435666393 904410736 692803997 -331825012 -227695083 -87180903 -548910596 103122752 188401195 843227376 -886354883 60512065 567660763 804939776 -2644789 -95283119 -356400350 -638227062 -672218217 672588138 124821925 105422737 84194237 -450215905 920775014 198726688 838144547 -353242670 60471513 -199237412 683383883 226223823 967957934 -13789375 622052510 -521202216 -853466799 -163382881 -200501408 592303545 -797112929 212188473 223103118 960626881 -828399458 453573548 -5133384 791606066 842365309 -520027342 -552504082 -758336402 -66517644 919651570 972549739 -321290184 -495593892 877144608 -160488379 548515464 -190404881 335287518 62636048 -295316565 640271156 -738118029 301230442 -942175373 128799385 -890076206 -728242464 -810007343 -719014278 372277975 733220601 579871583 652333560 463919320 151645559 302416918 -686064344 586088252 -194162288 874263661 -404204490 -958232633 469265801 -516846457 -794249997 -312657029 493805030 328135136 -861990216 -352824659 8756462 -655171717 747668448 -402744790 -880131841 -52507533 -998299623 -386158543 781542099 301128386 627266320 -395568325 -856328601 -300178748 165021277 36994199 442701610 299783863 -616750852 817274565 -236790540 -532192867 531288268 854776287 577255173 692101401 -723127910 -507497684 -553316646 17882316 -443028807 -90076150 -668611992 -121643827 -785670937 89716724 -711112045 990814775 21519977 63937938 -981962308 979512344 697082542 -648493512 -457022906 -466262524 337559725 384468605 -998820174 398617157 175723451 -488985464 -295210807 46346538 425337375 954480590 -717520215 -861546996 959804434 255564690 -263807914 698199829 -621741658 -155895385 -350123934 500415646 916796758 -625054868 645684456 818296006 6819769 -778218041 -87087718 -125966925 -181995108 200067633 350676069 954743286 440587707 297382595 793370893 -190938084 914507855 781151645 96222722 -540767991 -445334207 -440863270 -597172688 961931688 18059460 359051144 802183717 -407111624 490379945 240093606 19538595 -692557334 -590583454 -983271285 91806657 -863326118 158660340 190901495 37554223 77017256 5602059 -179867211 81913495 275550775 628755495 711080425 -251100386 672326132 567413243 -760950890 -839505224 263357728 988033149 -927613776 -754154010 385976706 1465438 -170569613 -266775089 -764943092 -488182695 -913379002 -319735176 -581153581 996507720 -943446138 897458468 -616894459 -797563535 314943318 940845094 940222130 259883846 39124265 435423532 -419465134 -44422088 -521144195 -614862182 -35318555 -860171366 -824808585 -905677712 635499478 -841722819 -26126725 819935608 -399055131 509822634 -613418274 910682376 112245029 947814652 -499243341 -484238475 282688545 -612664745 -82170942 707521300 -563788518 318450697 482743191 -20493330 -458736532 766952118 -573770753 -989069041 537153107 -618626504 726527210 130964672 534778302 576033055 410399077 -598413303 264015068 348707122 -28030055 452537872 -8017301 199960737 -910374889 -136376285 -966624562 917082972 -437143061 -166003268 -995135150 320680581 30414418 -512406721 -698332183 678707818 75044753 -79208522 -1932825 -196234683 -541364893 227404712 426896729 -431829543 137605949 -63312070 -721202670 -144004568 -682736792 898285662 -778135914 195322766 -478391863 855640744 -925220718 -783788887 -122963413 55704133 3446288 891003023 241680633 486447114 690601551 -103720115 -541277912 -497129346 -567289683 -866534008 121138574 414739191 -903809064 -506715972 -198868987 649086740 -642956590 468880923 -194585781 -662150739 -340496765 997650724 601968398 418712050 -858669048 422915177 297774199 218472791 -806385900 -760775932 222544056 827324793 -616623292 -54237359 78369469 -176433719 240309313 93614694 391230040 -644754998 879089097 -927233126 -159813781 114838758 727066249 -706610369 942917543 -738409670 -967466800 750587892 985514876 383634926 837960956 -122121998 682933599 22560133 -250766726 936209564 -294322805 576881071 -569796584 508230747 -617796706 510670106 216688961 826442747 -936669696 -342082297 -508710485 226715304 553764846 601801083 8585622 425997372 669041448 -305652749 -890467579 220544144 680580008 319962640 786447387 554934158 -205186121 939228054 -396482270 -377620326 987518296 31191788 577681998 -419547652 -926022580 360488960 612584567 -80827560 670042241 909108008 153906511 121899232 -148915588 316394451 677992684 -115467428 -122427829 393367612 -566198261 -316270555 -943428944 682598545 330565370 -582350868 -570711374 541046226 -953897750 386912548 4747161 955709960 957265190 -146361843 -894371659 714266176 311056952 455304498 255418742 835784222 55914580 -193379099 -651940121 876119640 -278986041 -820015244 514924115 -924199284 463528406 155196621 64047449 -964892804 896702947 920240505 -730880339 -823898352 289623548 -858432970 -84838882 704235091 13121407 -100382833 33689823 -656029256 43164594 -468103939 -947816021 686728662 569408130 -349730612 68014706 -15542296 794714959 -168410302 676235487 -324087472 444142846 -389037365 867629892 -708190242 115730558 -516302485 114511063 -987050088 47920488 814670451 384110310 83297612 -957325188 537399660 953143142 -910637554 -878188888 30529962 -710385045 981314899 512215263 -649721666 573022904 -228733625 -709047658 285877603 708732547 252502584 -131768289 -568137306 27560235 -296646411 -311532513 794049932 582964778 -259813235 -96004925 31535826 897462223 -589250432 579190119 -234621841 585995057 -690435384 838172626 -502393049 -399380069 -697291744 -22183043 242410824 -768414303 834804979 358177784 66315016 310815734 625610909 -865637466 -551763115 -374038286 999402427 -238842461 -861933327 -675313249 -88208322 382263497 -850763019 -464096596 510350035 960175961 492174232 856551132 -443738032 369811427 -9790670 589835850 207462613 605008734 158931436 -640153651 128902782 -727395753 -249302521 -66587018 -700492350 82717510 547095749 -587453130 246920555 936821757 613712888 -365493603 462771250 330727482 -958944731 -856090947 152778230 800492371 -300508519 -442735823 -441915274 -577628887 -764487561 -540020243 848727923 -963495465 599663748 371434806 694703268 540049921 496752733 -806863878 588884941 -734022693 631859112 -278931663 -333306118 70691232 998224587 103211028 -443300860 -464223714 390796405 -58278972 -472295976 -644948582 -565837563 955832 -950857869 -84616790 -518250915 750989341 145325325 70234288 534450407 -191124142 -902588685 -826570443 -290704007 -346592053 864227163 -760719331 693704429 372675932 923832777 125450784 -543332096 -768961562 -210895599 254569812 58170781 -313563636 557248149 -27062313 536008452 355562003 -954549889 775913356 312840855 -717465702 731235042 523499073 -917892566 -865448498 225864911 703154820 272304800 295050806 193301981 803916573 -77099931 -23952929 172909659 20879445 331746694 -132561888 30186932 226767591 330141854 437074321 333388508 73960585 -363549799 -881169239 -496678426 221665049 -233113452 -476169736 -293124430 637787237 -236800515 221910212 -175486825 -191077850 668227237 793178427 -505309788 -481396127 891122611 -289029393 -662711915 806029026 -167329317 -981688410 931164335 -334825100 475684709 689051544 -815637562 630285044 -659368793 317097674 -415393394 21490031 -970019804 870138165 701552765 -348419751 -920595395 693376918 711619791 -816713625 -710923769 345120465 670342307 -649892855 -575955144 -782547965 100947011 648742279 255692090 -245049565 -758825855 -16906825 -998250654 163567187 -786948804 -19547559 880549434 529556425 911758850 428472079 607954426 -13209516 857503440 -347575300 881013270 318915726 -267885836 -834892809 -971646991 -544505837 412965996 -145331537 269572773 -349034289 209355304 -973337650 -735768992 901034942 -714096904 377285613 976465740 -133750861 945749427 497725393 236808546 519719734 -946811348 347079158 784480029 482659394 280630626 -352392896 132889174 63589148 293068837 343356304 799958511 -792275439 121022879 -804993082 137869748 -674946613 985326405 -210068592 -27570720 -528215271 71204719 86839169 -843649511 864083858 738252305 573489957 106834268 -815106185 718537798 -896485189 -689015226 -714747113 110388424 -637198154 -779685595 791193027 382445081 -986325448 -404882720 823014399 -601146835 -324556740 408755176 -949607024 391112144 969083603 389075006 693984935 249943844 257653267 717463310 316361837 -155510333 492571407 327788833 -982229451 -977089103 775350878 389161878 814171124 -200587834 -650361290 314330440 30466828 -29721781 405648392 -716772594 -761948029 711750564 90532162 -637472310 28272611 -615706838 859567807 378769022 -715522638 -760827113 915280076 295837577 110267864 -42972594 719176660 -702747893 248079799 -432975068 980134693 2498361 -682681453 276056203 181204277 906612190 552377456 671732899 882428791 98814772 192035929 -833293535 -546186406 -286450644 296928332 581368348 402224525 -38739457 352669217 203680838 -69728172 446975534 762178967 -278074680 265046103 823024125 -248569935 565199917 17175808 -185385857 -803605366 876205815 -263689781 809074119 752397392 588239826 209120182 254774208 515990285 -655804827 588070891 -54134411 147205394 834505369 328117553 -684262632 201764819 118414696 385241437 -729653217 -961794465 -213342001 372573015 -61616759 69182330 271863005 522652616 -515696065 382280537 45795310 463765085 352411385 213155825 -915300933 240054255 -296631524 260247268 -662612993 -393329280 690834657 -323083167 -170759696 813341486 865324543 483444420 284713362 70449821 -338912575 -231824757 885809691 -919323951 -49634102 -700566212 860242514 463198644 -475480399 546507867 -878659272 -376583137 226413328 867881283 -989038389 680913156 757233449 -657462763 -328667737 -19591830 -957732774 740622141 981341898 609371654 -237760850 351211422 406640496 877049971 -170226032 -767271590 -432061155 -894654961 190862234 178330943 -275213383 -532315535 108101259 -49843503 370958805 -910393257 442426096 29312780 518803799 -948153255 -611317978 565094020 -349017280 993455012 -140776561 844883860 3791120 961395575 289087845 -669559169 567158162 -703889202 470233127 739839839 211460292 -734434539 -294539418 -702909155 793106363 745996280 -92867796 352900765 372666387 -307025323 -944843511 -838728714 269484304 559547987 -54737989 466990558 -721017770 -500032372 -859249799 785565989 -863135679 -459481498 574531804 -86675368 660008418 -317106817 388626539 -528769658 -744155066 332559553 943130143 -2746 520533415 -297450147 -154647816 -405336267 860744158 871907334 950316650 254765193 772345453 -30015783 -855588197 -614263718 368334632 422644977 735238612 173273199 -362901906 -317393999 -264350959 -122749797 558375538 -87125328 -258655080 591891891 -176392777 -533607358 379504321 252212778 -860715878 -415682401 -936189740 -264170999 452784563 496992783 -462150125 440642898 491325826 248660166 -877671767 810615574 -300999371 705231269 959974236 418334404 821455122 -875914506 734054081 760975170 133413037 593755874 -230269747 -838040640 187724887 759926114 -650514721 -779096300 419960671 856131406 720462070 854992546 -433701334 175658495 244633819 796428232 -501133336 115947606 -827965614 986666324 294842183 3035796 791275280 897238778 570299017 -918794895 -440982678 -873858118 -645905600 156865611 57202591 -220499875 842325929 -586152837 -622804802 310695843 -161415228 -602031050 -985500316 -762642614 -140792584 118538413 450670901 -223065782 -644469881 -66577848 999475951 857914155 220878264 999631158 631945515 -209674913 103746174 686537622 -483141141 -986611946 -212202511 30491297 170686416 -840109436 81377670 -480681344 475527264 -239844694 -59139526 -472450411 775063295 617066468 375416283 395442713 -187068992 974850932 -217135441 -80974650 627636001 -874233662 -222605930 794346826 -480811744 556075797 316844728 -492961968 -422748164 -384895525 -217764756 -629103598 -200415601 -472401530 472875275 929272288 -702528679 -465579198 31673729 44841918 392454538 -909762158 958247631 -379486198 92267551 -717170940 344854989 -961203035 614566136 -956718865 -706708022 -736302446 971704920 418248374 -154303193 -858505823 595410945 754425378 605877841 797308828 27393495 -859119093 -542309170 -322076770 595993693 212544651 264842683 639469022 -81898981 339791655 222845975 -202176982 783201552 851803815 377813748 -582704952 92394189 -968947088 303696375 631602256 315177267 175210545 -85857655 -584861943 898332038 703342823 867343431 -265671023 203561954 -462228446 486869422 -437982528 -923695072 936365863 -555997716 965618630 664414688 63613521 -125051486 -494615175 291115020 689430956 725229899 561741961 761436018 -262314243 -404753292 -983200592 -875592235 -183983057 -855328210 401071964 671523872 243127828 303009683 -359053576 166539339 741314971 3784025 132302166 489184700 -475869719 371542712 -787608029 -944661524 -548903098 402136639 -192536521 162356836 599870336 -747877425 -274539402 -786391232 -689607290 812271151 -595722095 -662378845 285277671 842245388 -793579273 -120804459 -303913388 -945270294 -25020590 160066639 -991462949 395539448 -455779340 -917188277 -316453062 896507060 823146373 -153751046 -77687080 364875483 -743451623 -322220031 -408553082 97878481 -875711367 264220228 -573996015 -78638508 -355448757 -300733339 334727295 -804753699 581652650 -751998443 328779590 531145286 937252870 385017855 -554521129 -526026214 32858449 643786398 749006108 -149332023 -265756516 -750765844 -514100387 440987284 -683662641 763323890 740405251 -8185555 -892949437 497750391 -311439637 460435052 965454366 -877510871 261192359 976414387 984951343 -416632403 862541110 927029346 -442107331 282618623 896804226 -498699335 -711812474 505131239 599042119 -610829903 -612709512 794458472 582688624 -351523148 451410155 -686698372 -406837403 -567185849 732382616 -982673784 41907617 359990907 -86383603 -281082605 799149348 150037038 256779907 179970978 306306001 667357056 -257662636 130016889 -464204793 -611962529 1132018 764253620 71617191 -260431829 421821188 284944548 -997556373 532875451 500229307 -325616449 -190436721 -977021346 -504897468 -996568793 968718988 909804140 -105074475 -289293313 467633031 -563406614 968463714 -74062423 272283251 -575793141 812854386 996287119 245873296 -702723651 -734356414 -790012944 503688501 999829435 -543594695 -70564583 554401993 393459948 134714202 415945304 -814359679 -835744831 47153815 -97288000 -609277589 66741034 80922549 925438654 -644285137 -469166659 -641732824 207692890 439634120 -364993153 -116870117 376408332 728706895 -767605876 -14389960 -473636365 -462632708 -754364525 -433007288 -227468795 -479074095 -37628734 -843729819 -432020944 -734414340 -313617496 -953453358 261849286 799551086 515661936 533949413 293958965 -737834329 323113890 -171664744 731158925 -954745061 103449589 -236871318 720095712 484632030 -989969012 -710135211 -182825735 605068906 -51496881 -468308952 7770924 974920670 -3315843 962039528 381840168 -436452505 -198949532 748996093 -125205175 93195962 -875206467 -244702323 680219837 684361121 790265574 313094597 -113624507 -62126059 450222969 -633099014 -107284819 -113409388 -73015049 644049495 -50941562 -670090977 -558490087 -536202034 366159527 287186087 -5676738 -55379703 -421366111 -47931296 -831091978 330679626 -335279767 -198441739 332168775 -688941837 -228261716 286385372 613976986 445593033 -124718141 -707239867 183406452 -392176272 897123113 -416865098 -60187669 -753497387 -919503350 696652600 -427248918 348056720 820161970 -748981915 -866801059 633552148 -2803277 -685279703 326869237 681246922 198997234 -157464978 -240955759 15289289 -266917413 -801064776 -971544268 505132981 624754049 932212035 -584294863 148861711 -479585577 -491822463 -327787686 -840734602 -810299120 233507009 -918071407 13030169 -268848390 340291644 564284434 726985875 -343194439 22379115 552106896 -526741624 -657668830 214222276 -715475618 -483094057 850948467 709130532 -988017043 -985683734 -499490532 -678071838 -294298428 285629100 -726561265 -274820242 -67020780 -450228107 524349994 67322863 498397894 -844495637 -114238197 -277946388 801443871 202121795 685121366 -760550521 -959654017 467546708 -869006682 -431587210 180214643 819419949 996913623 884447976 -843855230 835073004 848440437 -214196128 77297687 258526579 848034121 723517710 835241990 -730216961 -135624884 252298847 -82265314 943324756 118099082 -297499337 -911823393 487077765 -325373661 733057096 -275437460 362628900 446925007 -44953218 -646645198 -172854238 -157051798 -394755365 66641438 -874880945 650106314 -424767947 206744372 149490219 -910662232 -797810968 154828150 400531755 584947256 110147710 -746925147 -819364526 -445531207 -409827111 564234023 973256297 -693261266 -361155847 865411076 -27347749 -335001561 95976555 272138 -328157649 -69960008 -652105905 -199477231 719183650 -959553909 -723393788 -271534503 -314567328 527359346 -766171889 60455647 262628564 48160499 -647150097 609486041 671113193 -195659042 -52803484 -781907071 -997425486 -458870647 -153733457 -808219746 -276469284 650094578 847127259 452677090 -164472795 -117132025 -307491981 -784994686 -617355076 914571302 -124468854 245119578 457198555 503748779 977797147 -365674217 788674154 -762552048 -957578593 709455945 592433496 -302419991 -746946979 266938133 -311684791 -554150745 564253688 -256301804 51328979 428372061 -529481566 959661650 796884834 939259774 -329298363 518996485 -898357307 -991294862 -68070216 -400074666 36222317 178895451 -905198281 535248338 412394138 -572752472 -489028270 667590157 -549937249 142983253 419340337 -483443495 529124955 -511710928 -690073437 -925340879 124661291 -274036792 -276228370 -391223874 -777586041 529722622 -250445698 232841130 338879992 -284330293 567342136 -189626235 -532287775 -847052015 838683235 172514851 -479970854 748805764 633058669 447322925 -906851783 -468424671 -887397464 -454830710 218746244 511498400 261281489 383402014 966944531 -892809466 58059327 -355989836 -805676221 721409111 -162672219 -707181066 -413092780 -747558808 821426820 -223067751 882645728 -304592431 224746032 461675158 -196592647 416598785 -826649988 -818649198 -966445347 873014893 924317444 -652529313 -829559720 360539105 -423224040 933819803 -125982281 949846263 359809088 630419830 945506660 930655908 -502930063 200152722 -515407069 511257603 -192035352 520156606 -733498869 880459125 -869100206 574557129 -806771821 398203501 168821960 589126319 274941971 -487536728 -924808657 579436327 -142102827 83354783 270664261 678094415 997168803 490153772 -633446243 -142059744 -909549739 -287396026 193275191 -882941693 880953851 -950594328 936923579 -997006980 -989113950 328151569 237058130 -663293179 -115964217 -831173916 927800865 -787237130 206468950 12539375 139938659 165514399 886558045 694175368 -345652446 695756664 218492498 180640596 215448722 54859376 -433191683 424448737 -772368374 -548245919 328237059 -73299853 762227104 163146149 -276320495 -277341476 156160863 -762679614 153798817 828792708 283183400 -27062208 268386924 445481865 -107251678 -391727204 591748351 -276386849 810143061 345550477 -524251560 641903373 -700051421 -388525841 -387170849 -973150763 709356871 502531802 -239303841 -153325613 571460606 347666495 -406401568 931094031 351426023 -442653807 923651684 -814078076 -922170309 905264577 -238973067 -613836625 -867261689 -802463704 719842303 -422179893 316604802 -418751410 -897901740 -324738601 583215931 -685226687 -4301844 842546210 158021287 534001114 -66214673 -643275448 -185171509 338978617 -579196931 -402156942 670952067 349627845 788212146 323259435 -283262178 498870150 -358267165 919001132 -414286237 210502822 -783997118 -597784828 703433992 -66608028 -2438043 872404828 421997765 -719321311 -623474833 -433681579 -447754476 -406200424 193421149 603320845 854258056 -954379793 758695745 820044231 -445306536 -903972560 -399234292 -356350170 587446858 500481956 66261966 596765663 861387916 -408261362 -547111896 -738729999 -724937256 524711387 611757859 -960843883 362259214 337516334 -24693648 491751256 967684928 -859838071 -601899047 -777068054 -235588588 -391963016 -332076255 -534093249 10021518 -36283922 840048026 -567743074 -883925043 33369747 -181498706 920168227 -137865485 -544089186 409812221 -228999889 406518041 732895802 203668239 295958061 -337822456 -567534527 509560091 952709283 821587995 -552545227 554204249 710916030 -858629210 -139477571 856976458 191201791 -666438119 -188304000 786624500 329268505 -517373617 -655926840 205288939 621816880 396921300 88401599 -939039540 916510389 351862715 -703913883 443018450 -605045169 475202540 610137175 620657415 628930113 -840877792 -350771752 -17922782 -421613437 994696724 -11285931 -201787504 412076490 102731239 -269516106 -508273729 221930361 -487424523 998621719 -231431248 -818614508 46013431 312979649 -357084703 381802613 113547861 -571356257 -976752288 197084966 -775207571 -185503411 -574145637 -960174900 398670516 752576404 84248915 -789094351 -875876364 892010112 -687464273 -898836756 77016243 -357430496 -519564691 -381292625 267859892 -676621350 -540379085 634523437 -108782953 409380968 13436420 -154029658 109673883 -977502349 724238542 -86994160 140284874 -692204512 423751652 -384565865 464198463 975423618 -976283709 -803614434 623386554 -367873727 -473158438 690531915 -268398787 25581847 731887356 -633616137 356956928 -519597855 -153870316 -657639938 -328465798 607337809 -583336324 634742241 932924303 -152421007 917125564 422713179 -293819874 -909840424 -393774736 -537159262 -75845961 358719592 -418881465 -608585669 -903303574 839895232 850670887 -135968134 650933691 256814405 731626606 -893845355 -276629327 53772488
