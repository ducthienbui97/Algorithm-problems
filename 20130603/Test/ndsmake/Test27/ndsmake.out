1402215892367
